src/index.css

```diff
@@ -0,0 +1,74 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  
+  font-family: sans-serif;
+  margin: 3rem;
+  background-color: #dfdfdf;
+}
+
+h1, h2 {
+  color: #333333;
+}
+
+.btn {
+  font: inherit;
+  padding: 0.5rem 1.5rem;
+  cursor: pointer;
+  border-radius: 4px;
+  background-color: #800040;
+  color: white;
+  border: 1px solid #800040;
+  margin: 0 1rem;
+}
+
+.btn:hover {
+  background-color: #9c1458;
+  border-color: #9c1458;
+}
+
+.btn--alt {
```

Comment:
Interesting double-hyphen convention:
`btn--alt`
`btn-alt`
Make sure the team is on the same page about this.